app/javascript/components/Payment/Payment.js

```diff
@@ -52,6 +52,12 @@ export class Payment extends Component {
     };
   }
 
+  setRecurringCheckedForGermanPage() {
+    if (window.champaign.page.language_code == 'de') {
+      this.props.setRecurring(true);
```

Comment:
We have a `recurring_default` value in the plugin model: 

https://github.com/SumOfUs/Champaign/blob/6f4b77ede2b7f010507bcaaf4314cf437700ceb9/app/models/plugins/fundraiser.rb#L35

We can use that to set german pages to `recurring` in the backend. The fundraiser plugin (frontend) reads that value when it is initialised.

Reply:
@eyko @shivashankar-f22labs so this is already possible when creating a new page?

![image](https://user-images.githubusercontent.com/12949/66924138-fe25fc00-f021-11e9-8133-20d0851479fb.png)

Reply:
Yeah